maze_bench_nim.nimble

```diff
@@ -14,5 +14,5 @@ requires "nim >= 1.6.0"
 requires "pixie >= 3.0.4"
 
 task benchmark, "Runs the benchmark":
-  exec "nim r -d:danger --opt:speed --gc:arc src/maze_bench_nim.nim"
+  exec "nim r -d:danger --opt:speed --gc:arc --passC:-flto --passL:-flto src/maze_bench_nim.nim"
```

Comment:
Also a note for @avahe-kellenberger - `opt:speed` is always implied by `-d:release` or `-d:danger`, there's no need to specify it manually :)